prow/cluster/deck_private_rbac.yaml

```diff
@@ -1,6 +1,8 @@
 kind: ServiceAccount
 apiVersion: v1
 metadata:
+  annotations:
+    iam.gke.io/gcp-service-account: prow-control-plane@istio-testing.iam.gserviceaccount.com
```

Comment:
make sure you check/allow prow-control-plane workload identity mapping for each service that needs it.
```
  workload_members = [
    "serviceAccount:${local.project_id}.svc.id.goog[default/crier]",
    "serviceAccount:${local.project_id}.svc.id.goog[default/deck]",
    "serviceAccount:${local.project_id}.svc.id.goog[default/hook]",
    "serviceAccount:${local.project_id}.svc.id.goog[default/prow-controller-manager]",
    "serviceAccount:${local.project_id}.svc.id.goog[default/tide]"
  ]
```

Reply:
Good call. I am adding this to terraform